.travis.yml

```diff
@@ -47,4 +55,4 @@ matrix:
     - node_js: '9'
     - node_js: '7'
     - node_js: '5'
-    - env: TEST=true ESLINT=next
+    - env: TEST=true ESLINT=4
```

Comment:
let’s remove this line

Reply:
Whatever you say, fixed in 61372b88cec82c453d65091068b3d37afe3d4d25 :)